### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by iakoder**
Change __get (actually MongoCollection's method) to get.

### CODE REVIEW

File: standard/standard_3.php [hunk 1]
```diff
@@ -68,7 +68,7 @@ function convert_uudecode ($data) {}
  * @param mixed $number <p>
  * The numeric value to process
  * </p>
- * @return number The absolute value of number. If the
+ * @return float|int The absolute value of number. If the
```
Comment (andrey-sokolov) [inline standard/standard_3.php]: Merged this change in https://github.com/JetBrains/phpstorm-stubs/pull/256

**Comment by andrey-sokolov** [comment]
Please create a separate pull-request with the first commit. Pull request name should be based on commit you've included in it.

**Comment by iakoder** [comment]
@andrey-sokolov sorry, who/what are you asking?

**Comment by b1rdex** [comment]
@iakoder, @andrey-sokolov asks you to rebase your pull request, because c75e079 is already in tree. Also, you need to rename your pr to something like `rename mongo collection __get method`.

**Pull request closed without merging**